src/tools/compiletest/src/header.rs

```diff
@@ -1034,19 +1034,6 @@ impl Config {
         }
     }
 
-    pub fn find_rust_src_root(&self) -> Option<PathBuf> {
-        let mut path = self.src_base.clone();
-        let path_postfix = Path::new("src/etc/lldb_batchmode.py");
-
-        while path.pop() {
-            if path.join(&path_postfix).is_file() {
-                return Some(path);
-            }
-        }
-
-        None
-    }
```

Comment:
No need to use a heuristic to find source root, when we can just provide it through bootstrap.